duke/src/main/java/Deadline.java

```diff
@@ -0,0 +1,26 @@
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
why does this variable need to be protected instead of package-private when you are not subclassing it?